pkg/handlers/handlers.go

```diff
@@ -30,26 +30,34 @@ func About(w http.ResponseWriter, r *http.Request) {
 }
 
 func Contact(w http.ResponseWriter, r *http.Request) {
-	// recipient := ""
 	godotenv.Load()
+	recipientEmail := ""
+	recipientName := "Contact"
 
 	if r.Method == "POST" {
 		msg := models.Reqbody{
 			Name:     r.FormValue("name"),
 			Email:    r.FormValue("email"),
 			Question: r.FormValue("questions"),
-			Message:  r.FormValue("message"),
+			Message:  r.FormValue("Message"),
 		}
 
-		// switch msg.Question {
-		// case "events":
-		// 	recipient = " bporter@snohomishtribe.org"
-		// case "membership":
-		// 	recipient = "lloeber@snohomishtribe.org"
-		// default:
-		// 	recipient = "contact@snohomishtribe.org"
-		// }
-		// fmt.Println(recipient)
+		switch msg.Question {
+		case "Events":
+			recipientEmail = "bporter@devonfarm.xyz"
+			recipientName = "Events"
+		case "Membership":
+			recipientEmail = "lloeber@devonfarm.xyz"
+			recipientName = "Membership"
+		case "Language":
+			recipientEmail = "mevans@snohomishtribe.org"
+			recipientName = "Language"
+		// case "Programs":
+		// 	recipientEmail = "mevans@snohomishtribe.org"
+		// 	recipientName = "Programs"
```

Comment:
I think Lisa wants Programs to go to `lceniceros@snohomishtribe.org`.

Use `devonfarm.xyz` for the domain for both of Programs and Language right now so we can validate.

Reply:
I'm going to pull this branch and set up a single variable for the domain so its easy to change later